device/bgp.yaml

```diff
@@ -0,0 +1,88 @@
+components:
+  schemas:
+    Device.Bgp:
+      x-include:
+      - '../common/common.yaml#/components/schemas/Named.Object'
+      x-status: under-review
+      description: >-
+        Container of basic emulated BGP peer settings.
+
+        This container has the following optional sub containers.
+        Device.BgpAdvanced, Device.BgpCapability, Device.BgpSrTePolicy
+        
+        This container has the following optional lists of containers.
+        Device.Bgpv4RouteRange, Device.Bgpv6RouteRange
+      type: object
+      properties:
+        active:
+          description: >-
+            Activates the protocol which means the protocol can be started
+            or stopped.
+          type: boolean
+          default: True
+        router_id:
+          description: >-
+            The BGP router ID is a unique identifier used by routing protocols.
+            It is a 32-bit value that is often represented by an IPv4 address.
+          type: string
+          format: ipv4
+          default: 0.0.0.0
+        as_type:
+          description: >-
+            The type of BGP autonomous system.
+            External BGP is used for BGP links between two or more
+            autonomous systems (ebgp)
+            Internal BGP is used within a single autonomous system (ibgp).
```

Comment:
If type is changed to ebgp , the as_number_set_mode in this object should be changed to any mode except the default of do_not_include_as . eBGP requires some AS number to be advertised for session to come up, unlike iBGP.